src/OXRS_SENSORS.cpp

```diff
@@ -357,445 +316,547 @@ void OXRS_SENSORS::setCommandSchema(JsonVariant json)
     }
     _screenEnumNames.add("2 Lines of Custom Text");
 
-    JsonObject _oneOLED = json.createNestedObject("sensorOneOLED");
-    _oneOLED["type"] = "string";
-    _oneOLED["maxLength"] = 10;
-
-    JsonObject _twoOLED = json.createNestedObject("sensorTwoOLED");
-    _twoOLED["type"] = "string";
-    _twoOLED["maxLength"] = 10;
-
-    JsonObject _sleeping = json.createNestedObject("sensorSleepOLED");
-    _sleeping["type"] = "boolean";
-  }
-}
-
-void OXRS_SENSORS::conf(JsonVariant json)
-{
-  if (json.containsKey("sensorUpdateSeconds"))
-  {
-    _updateMs = json["sensorUpdateSeconds"].as<uint32_t>() * 1000L;
+    if (_ssd1306Found)
+    {
+      JsonObject _sleepOledenable = json.createNestedObject("sensorSleepOLEDEnable");
+      _sleepOledenable["type"] = "boolean";
+      _sleepOledenable["description"] = "Allows the screen to be put in sleepMode";
+    }
   }
 
-  if (json.containsKey("sensorScreenMode")) // for what mode the OLED is in
+  void OXRS_SENSORS::setCommandSchema(JsonVariant json)
   {
-    if (strcmp(json["sensorScreenMode"], "off") == 0)
-    {
-      _screenMode = OLED_MODE_OFF;
-    }
-    if (strcmp(json["sensorScreenMode"], "one") == 0)
-    {
-      _screenMode = OLED_MODE_ONE;
-    }
-    if (strcmp(json["sensorScreenMode"], "two") == 0)
-    {
-      _screenMode = OLED_MODE_TWO;
-    }
-    if (strcmp(json["sensorScreenMode"], "three") == 0)
-    {
-      _screenMode = OLED_MODE_THREE;
-    }
-    if (strcmp(json["sensorScreenMode"], "four") == 0)
+    if (_pcf8523Found)
     {
-      _screenMode = OLED_MODE_FOUR;
+      JsonObject _rtcItems = json.createNestedObject("sensorRTC");
+      _rtcItems["type"] = "array";
+      _rtcItems["description"] = "Ability to update the RTC time - year,month,day,hour,minute,seconds. ALL REQUIRED";

```

Comment:
Shouldn't need `ALL REQUIRED`? The JSON schema defines these as required so will validate for the user.